Failed to install with python 3.11

Here's the output after I type `pip install -U pywinpty`. I am using the python 3.11.0.

```
Collecting pywinpty
  Using cached pywinpty-2.0.8.tar.gz (23 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: pywinpty
  Building wheel for pywinpty (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for pywinpty (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [53 lines of output]
      Running `maturin pep517 build-wheel -i C:\Users\admin\AppData\Local\Programs\Python\Python311\python.exe --compatibility off`
         Compiling autocfg v1.1.0
         Compiling once_cell v1.15.0
         Compiling target-lexicon v0.12.4
         Compiling libc v0.2.137
         Compiling windows_x86_64_msvc v0.40.0
         Compiling proc-macro2 v1.0.47
         Compiling quote v1.0.21
         Compiling unicode-ident v1.0.5
         Compiling syn v1.0.103
         Compiling windows_x86_64_msvc v0.42.0
         Compiling parking_lot_core v0.9.4
         Compiling either v1.8.0
         Compiling smallvec v1.10.0
         Compiling cfg-if v1.0.0
         Compiling scopeguard v1.1.0
         Compiling indoc v1.0.7
         Compiling bitflags v1.3.2
         Compiling unindent v0.1.10
         Compiling num-traits v0.2.15
         Compiling lock_api v0.4.9
         Compiling memoffset v0.6.5
         Compiling windows v0.40.0
         Compiling which v4.3.0
         Compiling windows-sys v0.42.0
         Compiling pyo3-build-config v0.17.2
         Compiling pyo3-ffi v0.17.2
         Compiling pyo3 v0.17.2
         Compiling parking_lot v0.12.1
         Compiling winpty-rs v0.3.9
         Compiling pyo3-macros-backend v0.17.2
         Compiling enum-primitive-derive v0.2.2
         Compiling pyo3-macros v0.17.2
         Compiling pywinpty v2.0.8 (C:\Users\admin\AppData\Local\Temp\pip-install-3ehp_11p\pywinpty_9fce61f346de4556b8b9f82fb37d2d68)
      error: could not compile `pywinpty` due to 2 previous errors
      馃挜 maturin failed
        Caused by: Failed to build a native library through cargo
        Caused by: Cargo build finished with "exit code: 101": `cargo rustc --manifest-path Cargo.toml --message-format json --release --lib --`
      馃摝 Including license file "LICENSE.txt"
      馃嵐 Building a mixed python/rust project
      馃敆 Found pyo3 bindings
      馃悕 Found CPython 3.11 at C:\Users\admin\AppData\Local\Programs\Python\Python311\python.exe
      error: linking with `link.exe` failed: exit code: 1181
        |
        = note: "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.33.31629\\bin\\HostX64\\x64\\link.exe" "/DEF:C:\\Users\\admin\\AppData\\Local\\Temp\\rustcaeHmjJ\\lib.def" "/NOLOGO" "C:\\Users\\admin\\AppData\\Local\\Temp\\rustcaeHmjJ\\symbols.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.0.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.1.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.10.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.11.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.12.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.13.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.14.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.15.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.2.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.3.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.4.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.5.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.6.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.7.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.8.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.winpty.9047eca6-cgu.9.rcgu.o" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.nrzi735t35xlbn6.rcgu.o" "/LIBPATH:C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps" "/LIBPATH:C:\\Users\\admin\\.cargo\\registry\\src\\rsproxy.cn-8f6827c7555bfaf8\\windows_x86_64_msvc-0.42.0\\lib" "/LIBPATH:C:\\Users\\admin\\AppData\\Local\\Programs\\Python\\Python311\\libs" "/LIBPATH:C:\\Vim\\lib" "/LIBPATH:C:\\Vim\\vim90" "/LIBPATH:C:\\Users\\admin\\.cargo\\registry\\src\\rsproxy.cn-8f6827c7555bfaf8\\windows_x86_64_msvc-0.40.0\\lib" "/LIBPATH:C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libwinptyrs-2cb219541b8303db.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libwindows-84ae72de1d271cc3.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libbitflags-c59f221320647813.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libnum_traits-9c71a7b926a3bc85.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libpyo3-0d50053dd1c7892f.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libmemoffset-838cc49b0eec3042.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libparking_lot-2cc00fe0bda18a66.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libparking_lot_core-3e3a1db35b4b58fb.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libwindows_sys-ed5174040a4fc126.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libcfg_if-94f129e85b6df23d.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libsmallvec-fa6c20f10b44da3e.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\liblock_api-731e5e28694d85bd.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libscopeguard-59af7e2dc451f6a8.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libpyo3_ffi-f0554a1cc8221b18.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\liblibc-79e90d1cf9f97d28.rlib" "C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\libunindent-b159bd3b0660e8a1.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libstd-4a00450d169b6eb7.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libpanic_unwind-aebdd47781a8ded3.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_demangle-75d00692298b803c.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libstd_detect-212d184ebbb3da0c.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libhashbrown-bb2ae27a3860a710.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libminiz_oxide-aebeb1235b6ade9b.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libadler-8d905bd195def6d5.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_std_workspace_alloc-cdd36af8506e2d87.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libunwind-57642f47c953f1ca.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcfg_if-ddc101acd19cf3dd.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liblibc-304d7f6977c84a4b.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liballoc-32d7ee0be9e46c85.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_std_workspace_core-015e9ae63a74e3a1.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcore-15d7d0533432428b.rlib" "C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcompiler_builtins-cfe8447efe28cb1e.rlib" "winpty.lib" "windows.lib" "python311.lib" "legacy_stdio_definitions.lib" "advapi32.lib" "userenv.lib" "kernel32.lib" "ws2_32.lib" "bcrypt.lib" "msvcrt.lib" "/NXCOMPAT" "/LIBPATH:C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "/OUT:C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.dll" "/OPT:REF,ICF" "/DLL" "/IMPLIB:C:\\Users\\admin\\AppData\\Local\\Temp\\pip-install-3ehp_11p\\pywinpty_9fce61f346de4556b8b9f82fb37d2d68\\target\\release\\deps\\winpty.dll.lib" "/DEBUG" "/NATVIS:C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\intrinsic.natvis" "/NATVIS:C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\liballoc.natvis" "/NATVIS:C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libcore.natvis" "/NATVIS:C:\\Users\\admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libstd.natvis"
        = note: LINK : fatal error LNK1181: cannot open input file 'winpty.lib'



      error: aborting due to previous error


      Error: command ['maturin', 'pep517', 'build-wheel', '-i', 'C:\\Users\\admin\\AppData\\Local\\Programs\\Python\\Python311\\python.exe', '--compatibility', 'off'] returned non-zero exit status 1
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pywinpty
Failed to build pywinpty
ERROR: Could not build wheels for pywinpty, which is required to install pyproject.toml-based projects
```

--- a/.github/workflows/linux_sdist.yml
+++ b/.github/workflows/linux_sdist.yml
@@ -15,7 +15,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                PYTHON_VERSION: ["3.9"]
+                PYTHON_VERSION: ["3.10"]
         steps:
           - name: Checkout branch
             uses: actions/checkout@v3

--- a/.github/workflows/windows_build.yml
+++ b/.github/workflows/windows_build.yml
@@ -20,7 +20,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                PYTHON_VERSION: ["3.7", "3.8", "3.9", "3.10"]
+                PYTHON_VERSION: ["3.7", "3.8", "3.9", "3.10", "3.11"]
         steps:
             - name: Checkout branch
               uses: actions/checkout@v3

--- a/.github/workflows/windows_release.yml
+++ b/.github/workflows/windows_release.yml
@@ -15,7 +15,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                PYTHON_VERSION: ["3.7", "3.8", "3.9", "3.10"]
+                PYTHON_VERSION: ["3.7", "3.8", "3.9", "3.10", "3.11"]
         steps:
             - name: Checkout branch
               uses: actions/checkout@v3

